cli/cli.go

```diff
@@ -596,6 +599,23 @@ func Priam(args []string, defaultCfgFile string, infoW, errorW io.Writer) {
 						return nil
 					},
 				},
+				{
+					Name: "aws", Usage: "Use ID token to update credentials in the AWS CLI configuration file", ArgsUsage: "<aws-role-name>",
```

Comment:
Not sure what you mean. Are you saying we should specify the prepended arn in the usage message, or that we should not prepend and ask the user to pass in the whole thing? Do you know if that arn prefix is constant?